phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/engines/EnginesController.java

```diff
@@ -146,7 +146,7 @@ public EnginesController(EnginesView enginesView, RepositoryManager repositoryMa
                 this.repositoryManager.addCallbacks(
                         repositoryDTO -> this.enginesManager.fetchAvailableEngines(
                                 repositoryDTO,
-                                engines -> this.populateView(repositoryDTO, engines),
+                                engines -> Platform.runLater(() -> this.populateView(repositoryDTO, engines)),
```

Comment:
Why is this necessary?
I mean `populateView` uses `Platform.runLater` internally therefore it shouldn't be required here

Reply:
Hum, not all the content of populateView. showWait() for example

Reply:
Yes, but `showWait` shouldn't require `Platform.runLater`.

Reply:
Hmmm, interesting.
Then I would suggest to add a `Platform.runLater` around `showWait()`.
This in turn would make it unnecessary to add `Platform.runLater` around `populateView` and `forceViewUpdate`.

Reply:
Done